.github/workflows/4.a.1-generate-webapi-client-npm.yaml

```diff
@@ -0,0 +1,102 @@
+name: 4.a.1-Generate WebAPI client library for NPM
+
+on:
+  workflow_call:
+    inputs:
+      image:
+        required: true
+        type: string
+      apiver:
+        required: true
+        type: string
+      ref:
+        required: true
+        type: string
+
+jobs:
+  generate-npm-client:
+    runs-on: ubuntu-latest
+    services:
+      webapi:
+        image: ${{ inputs.image }}
+        ports:
+          - 8080:8080
+        options: >-
+          --health-cmd "curl -sS http://localhost:8080/health"
+          --health-interval 3s
+          --health-timeout 5s
+          --health-retries 5
+    permissions:
+      packages: write
+    env:
+      API: http://localhost:8080/api/v1/swagger.yaml
+    steps:
+      - name: Setup Node.js 18
```

Comment:
@YaSuenag can we try a more recent version (to avoid more painful updates in the future and ensure we stay up to date if possible)